client/asset/dcr/dcr_test.go

```diff
@@ -362,30 +367,31 @@ func TestAvailableFund(t *testing.T) {
 		t.Fatalf("expected locked = %d, got %d", tLotSize, bal.Locked)
 	}
 
-	lottaBit := tLotSize * 100
-	balanceResult.Balances[0].Spendable += float64(lottaBit) / 1e8
-	unspents = []walletjson.ListUnspentResult{littleUnspent, {
+	lottaOrder := tLotSize * 100
+	// Add funding for an extra input to accommodate the later combined tests.
+	lottaFunds := calc.RequiredOrderFunds(lottaOrder, 2*dexdcr.P2PKHInputSize, tDCR)
```

Comment:
This was a little counterintuitive, so I figured I would mention it. Combining `littleFunds + lottaFunds` is not enough to cover `extraLottaOrder = littleOrder + lottaOrder`, even though `littleFunds` is enough to cover `littleOrder` and `lottaFunds` is enough to cover `lottaOrder`. 

In English: you must prove ownership of fewer funds in order to place two one lot orders than you would to place one two lot order with two inputs. 

In Go:
```go
func TestThis(t *testing.T) {
	one := calc.RequiredOrderFunds(tBTC.LotSize, dexbtc.RedeemP2PKHInputSize, tBTC)
	two := calc.RequiredOrderFunds(2*tBTC.LotSize, 2*dexbtc.RedeemP2PKHInputSize, tBTC)
	fmt.Println("one =", one, ", two =", two, ", two <= one*2 ?", two <= one*2)
}
```
will print `one = 1007650 , two = 2020366 , two <= one*2 ? false`.